### PR TITLE
Add package xenon

### PR DESCRIPTION
**Pull request by windowsboy111**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -20058,5 +20058,19 @@
...
+    "method": "git",
+    "tags": [
+      "wxwidgets",
+      "nim",
+      "webview",
+      "browser"
+    ],
+    "description": "an efficient, simple, and elegant browser.",
+    "license": "GPL-3.0",
+    "web": "https://windowsboy111:Chan Ho Long, Cyrus@github.com/windowsboy111/xenon"
```
Comment (ba0f3) [inline packages.json:20074]: I think you were accidentally exposed your github password

Comment (windowsboy111) [reply packages.json:20074]: oh freax lmao

Comment (windowsboy111) [reply packages.json:20074]: Idk how my password gets there but can you help me to rm it? thx

**Comment by ba0f3** [comment]
Change your password asap10:14, November 1, 2020, matriz <notifications@github.com>:
@windowsboy111 commented on this pull request.



In packages.json:
> @@ -20058,5 +20058,19 @@
     "description": "SSH, SCP and SFTP client for Nim",
     "license": "MIT",
     "web": "https://github.com/ba0f3/ssh2.nim"
+  },
+  {
+    "name": "xenon",
+    "url": "https://windowsboy111:Chan Ho Long, Cyrus@github.com/windowsboy111/xenon",
+    "method": "git",
+    "tags": [
+      "wxwidgets",
+      "nim",
+      "webview",
+      "browser"
+    ],
+    "description": "an efficient, simple, and elegant browser.",
+    "license": "GPL-3.0",
+    "web": "https://windowsboy111:Chan Ho Long, Cyrus@github.com/windowsboy111/xenon"

oh freax lmao

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Sent from Yandex.Mail for mobile

**Comment by windowsboy111** [comment]
changed already, thx so much for letting me know this stupid flaw.

File: packages.json (hunk 1 above)
Comment (narimiran) [reply packages.json:20074]: While at it, can we change `web` field to be a simple `https://github.com/windowsboy111/xenon`?

**Comment by hlaaftana** [comment]
This does not use Nim anymore nor has a Nimble file

**Comment by narimiran** [comment]
> This does not use Nim anymore nor has a Nimble file

Indeed. Closing.

**Pull request closed without merging**